apache.Dockerfile

```diff
@@ -16,7 +17,7 @@ RUN yarn --network-timeout 600000 && yarn install
 COPY . /dashboard/
 RUN yarn compile
 
-FROM docker.io/httpd:2.4.43-alpine
+FROM docker.io/$TARGETARCH/httpd:2.4.43-alpine
```

Comment:
why do we need targetarch ?

Reply:
build fails for `linux/arm64v8` if I don't specify `$TARGETARCH`.